guides/common/modules/con_ipv4-and-ipv6-requirements.adoc

```diff
@@ -0,0 +1,44 @@
+[id="ipv4-and-ipv6-requirements_{context}"]
+= IPv4 and IPv6 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+
+ifdef::satellite[]
+[NOTE]
+====
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+====
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure that an IPv6 loopback is configured.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+ifdef::satellite[]
++
+For a supported way to disable the IPv6 protocol, see link:https://access.redhat.com/solutions/5045841[How do I disable the IPv6 protocol on Red Hat Satellite and/or Red Hat Capsule server?] in Red{nbsp}Hat Knowledgebase.
+endif::[]
+
+The following requirements apply to installations in an IPv6 network:
```

Comment:
There are quite a few specific provisioning limitations. I wonder if we should mention those in the installation guide already or instead mention them in the provisioning guide. Then include a note here that there are provisioning specific limitation.

Reply:
True, but on the other hand it's also good to remind admins who set up provisioning because they may not remember from installing it or it changed since. That's why I think it's good to have it at least in the provisioning guide.

Reply:
To decide which would work best here, one would need to take a look at the requirement list here and at the provisioning guide. That's not something I should get involved in right now but since Avital is planning a redesign of the installation guide, I summed up this comment thread in her Jira epic: https://issues.redhat.com/browse/SAT-23920